ktor-io/common/src/io/ktor/utils/io/bits/MemoryFactory.kt

```diff
@@ -12,6 +12,11 @@ import kotlin.contracts.contract
  * By default, if neither [offset] nor [length] specified, the whole array is used.
  * An instance of [Memory] provided into the [block] should be never captured and used outside of lambda.
  */
+/**
+ * TODO: Solve design problems
+ * 1. length has no default
```

Comment:
Length had no default because of the compiler bug prevented me from making the default with expect/actual. Could be already fixed.

Reply:
added to comment